scene/gui/color_picker.cpp

```diff
@@ -966,9 +966,9 @@ bool ColorPicker::is_deferred_mode() const {
 void ColorPicker::_update_text_value() {
 	bool text_visible = true;
 	if (text_is_constructor) {
-		String t = "Color(" + String::num(color.r) + ", " + String::num(color.g) + ", " + String::num(color.b);
+		String t = "Color(" + String::num(Math::snapped(color.r, 0.001)) + ", " + String::num(Math::snapped(color.g, 0.001)) + ", " + String::num(Math::snapped(color.b, 0.001));
```

Comment:
Just curious, does something speak against just passing 3 as second argument to `String::num(double p_num, int p_decimals)`?

Reply:
I didn't know `num()` accepted a second decimals argument. I guess we can use that :slightly_smiling_face: